pkg/binauthz/binauthz.go

```diff
@@ -0,0 +1,39 @@
+package binauthz
```

Comment:
How about put this file in `pkg/kritis` (e.g., `pkg/kritis/binauthz/binauthz.go`) ?

Reply:
OK. I'm going to move it.

Reply:
Fixed in 69bd491